wezterm-gui/src/selection.rs

```diff
@@ -52,7 +52,7 @@ pub struct SelectionRange {
 }
 
 fn is_double_click_word(s: &str) -> bool {
-    match s.len() {
+    match s.chars().count() {
```

Comment:
if you change line 8 to:

```rust
use wezterm_term::{SemanticZone, StableRowIndex, unicode_column_width};
```

then I think this can be changed and support graphemes
```suggestion
    match unicode_column_width(s) {
```

Reply:
although my suggested changes might not be sufficient: not sure what it does for eg: double-width graphemes or longer combining sequences.
If it doesn't obviously work, then I'll take you original PR :)